File: sgifd-backend/src/main/java/com/sgifdbackend/demo/parametrage/entites/NatureAssistance.java
package com.sgifdbackend.demo.parametrage.entites;

import com.sgifdbackend.demo.entitiesBase.EntityBaseBean;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

import javax.persistence.*;
import java.io.Serializable;
import java.util.Date;

@Entity
@Table(name = "t_NatureAssistance")
@EntityListeners(AuditingEntityListener.class)
public class NatureAssistance extends EntityBaseBean implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String libelle;
    
    private Boolean isAppui;
    

	public Boolean getIsAppui() {
		return isAppui;
	}

	public void setIsAppui(Boolean isAppui) {
		this.isAppui = isAppui;
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getLibelle() {
		return libelle;
	}

	public void setLibelle(String libelle) {
		this.libelle = libelle;
	}

}

File: sgifd-frontend/src/app/models/NatureAssistance.ts
export class NatureAssistance {
  id: number;
  libelle:	string;
  isAppui: boolean;
  createBy:	string;
  deleteBy:	string;

  constructor(id: number, libelle: string, isAppui: boolean, createBy: string, deleteBy: string) {
    this.id = id;
    this.libelle = libelle;
    this.isAppui = isAppui;
    this.createBy = createBy;
    this.deleteBy = deleteBy;
  }
}
